.gitlab-ci.yml

```diff
@@ -0,0 +1,36 @@
+variables:
+  CURRENT_CI_IMAGE: v7679051-11c779a8
+  KUBERNETES_SERVICE_ACCOUNT_OVERWRITE: lemur
+
+stages:
+  - build-stage-image
+  - build-prod-image
+
+build-stage-image:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/images:$CURRENT_CI_IMAGE
+  stage:build-stage-image
+  when: on_success
+  # the 3h timeout is required for images compiling large projects for ARM using qemu
+  # we can revert back to 1h when we use native builders for ARM
+  timeout: 3h
+  tags: ["arch:amd64"]
+  variables:
+    CI_ENABLE_CONTAINER_IMAGE_BUILDS: "true"
+  script:
+    - cd publish&& docker buildx build --build-arg CI_COMMIT_SHA=${CI_COMMIT_SHA} --tag 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/lemur:v$CI_PIPELINE_ID-$CI_COMMIT_SHORT_SHA --push .
+
+# build a prod image when we create a new tag
+build-prod-image:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/images:$CURRENT_CI_IMAGE
+  stage: build-prod-image
+  when: on_success
+  # the 3h timeout is required for images compiling large projects for ARM using qemu
+  # we can revert back to 1h when we use native builders for ARM
+  timeout: 3h
+  tags: ["arch:amd64"]
+  variables:
+    CI_ENABLE_CONTAINER_IMAGE_BUILDS: "true"
+  script:
+    - cd publish && docker buildx build --build-arg CI_COMMIT_SHA=${CI_COMMIT_SHA} --tag 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/lemur:$CI_COMMIT_TAG --push .
+  only:
+    - tags
```

Comment:
Ultra nit: missing newline at end of file

Reply:
shoot, I had added newlines to those files but then forgot to push from my local branch. Files have been updated